### ISSUE TITLE
Integration cannot be load

### ISSUE THREAD

**Issue by Guddy87** [frodef/nilan-cts600-homeassistant#9]
Hello

I get this error

gger: homeassistant.setup
Quelle: setup.py:466
Erstmals aufgetreten: 14:56:13 (3 Vorkommnisse)
Zuletzt protokolliert: 14:56:13

Unable to prepare setup for platform 'nilan_cts600.climate': Platform not found (cannot import name 'computeCRC' from 'pymodbus.utilities' (/usr/local/lib/python3.12/site-packages/pymodbus/utilities.py)).
Unable to prepare setup for platform 'nilan_cts600.sensor': Platform not found (cannot import name 'computeCRC' from 'pymodbus.utilities' (/usr/local/lib/python3.12/site-packages/pymodbus/utilities.py)).
Unable to prepare setup for platform 'nilan_cts600.button': Platform not found (cannot import name 'computeCRC' from 'pymodbus.utilities' (/usr/local/lib/python3.12/site-packages/pymodbus/utilities.py)).

Can youz help me with this?

![image](https://github.com/frodef/nilan-cts600-homeassistant/assets/117725011/f0757b4a-d721-45ea-81a0-e2b01e5ce6f9)
Startup time when it works is 44 secounds or more.

Always when the integration work then a other will not work (Solax Modbus). This issue are after i change from a Intel NUC gen3 to a gen7.

Version | core-2024.3.1
-- | --
Installationstyp | Home Assistant OS
Entwicklung | false
Supervisor | true
Docker | true
Benutzer | root
Virtuelle Umgebung | false
Python-Version | 3.12.2
Betriebssystemfamilie | Linux
Betriebssystem-Version | 6.6.20-haos
CPU-Architektur | x86_64
Zeitzone | Europe/Vienna
Konfigurationsverzeichnis | /config

**Comment by frodef** [comment]
Hi, I just got this error too after upgrading to HA 2024.3.2.

Why did you close the issue?

**Comment by Guddy87** [comment]
Hi. My issue was that i used Solax Modbus to and in the manifest file was this : "requirements": ["pymodbus>=3.6.6"],

I have changed rhis to : "requirements": ["pymodbus==3.5.2"],

Then both integration work with pymodbus.

I am not a specialist in python and integrations. I think that HA can not load twobdigferent pymodbus version. version 3.5.2  is a version where both can work. I have set this in both manifest.json files. If i set this only in the solax modbus integration than both work only when solax was started first


I hope this help. Maybe you use also a secound modbus integration
BR
